commit eff102460dc40c35aeb0013abe3e8b866cf990c1
Author: Ken Gaillot <kgaillot@redhat.com>
Date:   Wed Mar 16 17:51:32 2016 -0500

    High: build: fix scalability regression
    
    This reverts commit 0fe7a4d, which inadvertently negated the LIBS changes made
    by AC_CHECK_LIB. That causes the check for qb_ipcs_connection_get_buffer_size()
    to fail even when the function is available, which in turn disables code
    allowing the CIB to scale to large sizes.

diff --git a/configure.ac b/configure.ac
index 4c0b7a1..28bddaf 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1039,15 +1039,9 @@ if test x${enable_no_stack} = xyes; then
     SUPPORT_CS=no
 fi
 
-SAVE_CPPFLAGS="$CPPFLAGS"
-SAVE_LIBS="$LIBS"
 PKG_CHECK_MODULES(libqb, libqb >= 0.13, HAVE_libqb=1, HAVE_libqb=0)
-CPPFLAGS="$CPPFLAGS $libqb_CFLAGS"
-LIBS="$LIBS $libqb_LIBS"
 AC_CHECK_HEADERS(qb/qbipc_common.h)
 AC_CHECK_LIB(qb, qb_ipcs_connection_auth_set)
-CPPFLAGS="$SAVE_CPPFLAGS"
-LIBS="$SAVE_LIBS"
 
 LIBQB_LOG=1
 PCMK_FEATURES="$PCMK_FEATURES libqb-logging libqb-ipc"
